src/cli_shared/cli/client.rs

```diff
@@ -72,8 +71,6 @@ pub struct Client {
         |g| Duration::milliseconds(i64::arbitrary(g))
     )))]
     pub token_exp: Duration,
-    /// Display progress bars mode. Auto will display if TTY.
-    pub show_progress_bars: ProgressBarVisibility,
```

Comment:
This change is not backward compatible. Let's bump the major version number.

Reply:
done